ccdproc/tests/test_ccdproc.py

```diff
@@ -481,6 +481,47 @@ def test_flat_correct_min_value(ccd_data):
     assert flat_orig_data is not flat.data
 
 
+# test for flat correction
```

Comment:
I guess this was copied from the test above. I don't think this accurately describes the test case, it probably needs a `with mean_value` appended or we could remove it here altogether.